components/number-card.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types'
+
+import theme from '@/styles/theme'
+
+function NumberCard({label, number, color, size}) {
+  return (
+    <div className='number-card'>
+      <div>{number}</div>
+      <div>{label}</div>
+
+      <style jsx>{`
+        .number-card {
+          background: ${color === 'primary' ? theme.backgroundGrey : theme.backgroundWhite};
+          border: ${color === 'primary' ? 'none' : `${theme.primary} 3px solid`};
+          border-radius: 5px;
+          font-weight: bold;
+          padding: ${size === 'regular' ? '2em' : '.5em 0'};
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          gap: ${size === 'regular' ? '1.5em' : '5px'};
+        }
+
+        .number-card div:first-child {
+          color: ${theme.primary};
+          font-size: ${size === 'regular' ? 'xx-large' : 'x-large'};
+        }
+
+        .number-card div:last-child {
+          font-size: ${size === 'regular' ? 'x-large' : 'medium'};
+        }
```

Comment:
On pourrait simplifier en isolant le cas particulier comme ceci :
```suggestion
        .number-card div {
          font-size: ${size === 'regular' ? 'x-large' : 'medium'};
        }
        
        .number-card div:first-child {
          color: ${theme.primary};
        }
```

Le mieux étant tout de même de donner une classe spécifique à cette première `div`. Les `:first` et `:last` servent surtout quand il n'y a pas d'autre moyen d'atteindre le `div` autrement.